FrontEnd/src/PostFullDisplay.jsx

```diff
@@ -34,7 +51,7 @@ function PostFullDisplay({ userUID, post, isShown, onClose }) {
               </p>
               <button onClick={handleBookingFormOpen}>Book Now</button>
               <span>
-                <i className="fa-regular fa-heart"></i>
+                {favorited ? (<i className="fa-solid fa-heart hearted" ></i>) : (<i className="fa-regular fa-heart" onClick={()=>handleNotification(1)}></i>)}
```

Comment:
Nit: when there's no content, self-closing tag is preferred:
`<i className="fa-solid fa-heart hearted" />`